.gitignore

```diff
@@ -19,3 +19,4 @@ env/
 venv/
 .hypothesis/
 .python-version
+.dmypy.json
```

Comment:
I've added this because I'm using neovim with pylsp-mypy (with the dmypy daemon, which can create this file).

Reply:
No objections on this.

Note that you can include this in `~/.config/git/ignore` and git will ignore the file for all your local repositories.

Reply:
Ah thanks, did not know that. Good to know :)